util/reggen/reg_block.py

```diff
@@ -1,7 +1,6 @@
 # Copyright lowRISC contributors.
 # Licensed under the Apache License, Version 2.0, see LICENSE for details.
 # SPDX-License-Identifier: Apache-2.0
-
```

Comment:
nit: it looks like there are quite a few formatting changes in this PR that are unrelated to the feature... 
I would recommend to split such changes into a separate commit to help ease the review in the future.